lib/utilities.js

```diff
@@ -139,12 +141,19 @@ function getRootObject(memberExpression) {
   return currentObject;
 }
 
+function docsUrl(filename) {
```

Comment:
I presume this does support our custom rules with prefixes. `typescript`, `webpack`, `jest` and any others we may add in the future.

Reply:
Yeah I meant to just send in the prefix into the function. So `docsUrl('jest/no-vague-title')` makes sense as a "rule name" to me.

Reply:
ou, sneaky but clever. works for me